ui/src/containers/CreateVolume.js

```diff
@@ -9,6 +9,8 @@ import {
 } from '../ducks/app/volumes';
 import { fontSize, padding } from '@scality/core-ui/dist/style/theme';
 
+// We might want to do a factorization later for
```

Comment:
Comments tends to be forgotten, if we want to keep track of this it's better to create a debt issue :slightly_smiling_face:

Reply:
It's currently not really debt. Since we only have 2 forms in the UI it's too early to do some factorizations. However we can already see some component that might be a good candidate  to be reuse so we add this comment as a reminder.

Reply:
Fair enough, then a comment is enough for now.